roles/virt-install/files/vm-4.ks

```diff
@@ -0,0 +1,62 @@
+#platform=x86, AMD64, or Intel EM64T
+#version=DEVEL
+# Keyboard layouts
+keyboard 'us'
+# Root password is redhat
+rootpw --iscrypted $1$UIw76XGR$Y0zGLekqa9PR60TZV1O1l/
+# System language
+lang en_US
+# System timezone
+timezone America/New_York --isUtc
+# Use graphical install
+graphical
+# Network information
+network  --bootproto=dhcp --device=eth0
+# System authorization information
+auth  --useshadow  --passalgo=sha512
+# Firewall configuration
+firewall --disabled
```

Comment:
probably not a good 'example' to have firewall and selinux (below) disabled - please keep them enabled